drm/drm_debugfs_crc.c

```diff
@@ -75,6 +75,22 @@ static int crc_control_open(struct inode *inode, struct file *file)
 	return single_open(file, crc_control_show, crtc);
 }
 
+void *memdup_user_nul(const void __user *src, size_t len)
```

Comment:
this should be moved to the right place but iirc the memdup stuff was outside `gplv2` in the regular `linuxkpi`?